addons/hr_timesheet/__manifest__.py

```diff
@@ -59,7 +59,8 @@
         'project.webclient': [
             'hr_timesheet/static/src/services/**/*',
             'hr_timesheet/static/src/components/**/*',
-            'hr_timesheet/static/src/scss/timesheets_task_form.scss'
+            'hr_timesheet/static/src/scss/timesheets_task_form.scss',
+            'hr_timesheet/static/src/views/fields/progress_bar/project_task_progress_bar_field.js',
```

Comment:
If we move the file in `components` folder then we no longer need that change. 🙂